src/orders/components/OrderFulfilledProductsCard/ExtraInfoLines.tsx

```diff
@@ -0,0 +1,103 @@
+import { makeStyles, TableCell, TableRow, Typography } from "@material-ui/core";
+import { getStringOrPlaceholder } from "@saleor/misc";
+import { FulfillmentStatus } from "@saleor/types/globalTypes";
+import classNames from "classnames";
+import React from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { FormattedMessage } from "react-intl";
+
+import { OrderDetails_order_fulfillments } from "../../types/OrderDetails";
+
+const useStyles = makeStyles(
+  theme => ({
+    infoLabel: {
+      display: "inline-block"
+    },
+    infoLabelWithMargin: {
+      marginBottom: theme.spacing()
+    },
+    infoRow: {
+      padding: theme.spacing(2, 3)
+    }
+  }),
+  { name: "ExtraInfoLines" }
+);
+
+const messages = defineMessages({
+  fulfilled: {
+    defaultMessage: "Fulfilled from: ",
```

Comment:
Translation strings shouldn't contain whitespaces at the beginning/end - let's fix that in the refactor afterwards